lib/Zadm/Schema/base.pm

```diff
@@ -12,7 +12,7 @@ has schema => sub($self) {
     return {
     zonename    => {
         description => 'name of zone',
-        validator   => $self->sv->regexp(qr/^[-\w]+$/, 'zonename not valid'),
+        validator   => $self->sv->regexp(qr/^.+$/, 'expected a string'),
```

Comment:
hmmm are you allowing zone names consisting of space on purpose? how about `^\S(.*\S)?$`

Reply:
good point, thanks